_posts/2018-08-19-openfaas-on-fargate.md

```diff
@@ -0,0 +1,9 @@
+## Architecture
```

Comment:
Outline seems good. I would suggest starting with a bit of "Why is this needed?"